Lua strings reverse (#3156)
Codecademy/docs: content/lua/concepts/strings/terms/reverse/reverse.md

---
Title: '.reverse()'

> Description: ' rearranges a provided string in reverse order'

Reviewer: ```suggestion
Description: 'Returns a provided string in reverse order.'
```

Subjects:
  - 'Computer Science'
  - 'Game Development'
Tags:
  - 'Strings'
  - 'Functions'
CatalogContent:
  - 'learn-lua'
  - 'paths/computer-science'
---

The **`.reverse()`** function takes a valid string and rearranges it in reverse order.

## Syntax

```lua
string.reverse(originalString)

> 

Reviewer: remove empty line

```

> The **`originalString`** is the string to be reversed.

Reviewer: ```suggestion
The `originalString` is the string to be reversed.
```

Reviewer: bold is only applied to the first instance of the method or concept in the entry

## Example

> Consider the following example:

Reviewer: ```suggestion
The code below demonstrates a basic implementation of the `.reverse()` method:
```

```lua
originalString = "Hello, World!"
newString = string.reverse(originalString)

print(newString)
```

> Expected output:

Reviewer: ```suggestion
This will return the following output:
```

```shell

> "!dlroW ,olleH"

Reviewer: ```suggestion
!dlroW ,olleH
```
